R/customVarInfo.R

```diff
@@ -318,7 +318,7 @@ defaultVarInfo <- list(
   # scalar
   list(
     name = 'Scalar',
-    doesApply = function(v) is.atomic(v) && length(v) == 1,
+    doesApply = function(v) is.atomic(v) && length(v) == 1 && is.null(attributes(v)),
     hasChildren = FALSE,
     toString = function(v) paste(deparse(v), collapse = '\n')
```

Comment:
In this case, `toString = paste0(deparse(v), collapse = '\n')` should be enough. The only case that `deparse(v)` may return multi-element character vector will be a very long string I guess.

Reply:
Does `paste(...)` have any side effects in the normal case? If not, I would leave it in there, just to be sure, since there seem to be some edge cases that are regarded as atomic (https://stat.ethz.ch/R-manual/R-devel/library/base/html/is.recursive.html) and the performance impact should be minimal

Reply:
Agree. Maybe `paste0` is more suitable as it does not add spaces to the original strings.